src/components/overlays/TheCreateNewTacticOverlay.vue

```diff
@@ -66,28 +65,26 @@ import Component, { mixins } from 'vue-class-component'
 import { EventBus } from '@/event-bus'
 import { RoomGetters } from '@/store/modules/room'
 import { Action, Getter } from 'vuex-class'
-import { Api } from '@/store/modules/types'
+import { Api, Collection } from '@/store/modules/types'
 import { StageActions } from '@/store/modules/stage'
 import { CustomStageConfig } from '@/util/PointerEventMapper'
 import { CanvasAction } from '@/store/modules/canvas'
 import { CanvasElement, CanvasElementHistory } from '@/types/Canvas'
 import TacticWatcher from '@/mixins/TacticWatcher'
 import uuid from 'uuid'
-import { AuthenticationGetters, JWT } from '@/store/modules/authentication'
+import { AuthenticationGetters } from '@/store/modules/authentication'
+import { TacticGetters } from '../../store/modules/tactic'
 
 @Component({
   name: 'TheCreateNewTacticOverlay',
   data: () => ({
-    rules: {
-      length: (v: string) => (v || '').length >= 1 || 'Name can not be empty'
-    },
     tactic: {
       map: {
         name: '',
         icon: '',
         desc: '',
-        width: 1,
-        height: 1,
+        width: 0,
```

Comment:
this should be 1 at default

Reply:
This was from your code so I will put it back :)